### PR TITLE
Replace travis with GitHub action

### LINKED ISSUE #119: Replace travis.org with travis.com or github-actions
Building on travis.org is ceased, so no tests are run on pull-requests and such.

Either this repo should be switched over to travis.com (must be done by the repo owner I guess) or to github actions which would require a PR with the workflow files.

### PR DESCRIPTION
**Pull request by rejas**
Fixes #119

### CODE REVIEW

**Comment by rejas** [comment]
so this PR would be ready to be merged @ben-eb. only thing questionable is the coverage part which would need a secret token generated (which only @ben-eb can do)

File: .github/workflows/node-ci.js.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+          npm ci
+          npm test
+          npm run coverage
+
+        # Run the coveralls action which uploads the
+        # lcov.info file
+      - name: Coveralls Parallel
+        uses: coverallsapp/github-action@master
+        with:
+          github-token: ${{ secrets.github_token }}
```
Comment (ben-eb) [inline .github/workflows/node-ci.js.yml:40]: Does this want to be `secrets.GITHUB_TOKEN` ? https://docs.github.com/en/actions/security-guides/automatic-token-authentication

**Review by ben-eb — COMMENTED** [review]
Thanks!

If it's needed I've regenerated the repo token for coveralls and put it in the `COVERALLS_REPO_TOKEN` secret.

File: .github/workflows/node-ci.js.yml (hunk 1 above)
Comment (rejas) [reply .github/workflows/node-ci.js.yml:40]: I assume yes. Not sure if the uppercase matters, but I can change that if you think so.

Comment (ben-eb) [reply .github/workflows/node-ci.js.yml:40]: > I assume yes. Not sure if the uppercase matters, but I can change that if you think so.

I think it does yes, if you could change please it that'd be great! 👍

**Review by ben-eb — APPROVED** [review]
Thanks! 🚀

**Pull request merged**